fdbserver/RestoreLoader.actor.cpp

```diff
@@ -216,6 +218,7 @@ ACTOR Future<Void> handleSendMutationsRequest(RestoreSendMutationsToAppliersRequ
 ACTOR Future<Void> sendMutationsToApplier(Reference<RestoreLoaderData> self, VersionedMutationsMap* pkvOps,
                                           bool isRangeFile, Version startVersion, Version endVersion, int fileIndex) {
 	state VersionedMutationsMap& kvOps = *pkvOps;
+	state VersionedMutationsMap::iterator kvOp = kvOps.end();
```

Comment:
i think this can be initialize to `kvOps.begin()`.  Thus line 242 (line 248's initialization) can be removed.